lib/types/application/entity.d.ts

```diff
@@ -50,8 +45,7 @@ type ClientSample = Record<string, string | number | boolean | null>;
 
 export interface Entity<F extends BaseField = Field, R extends BaseRelationship = never>
   extends Omit<Required<BaseEntity<F>>, 'relationships'>,
-    ServerEntity,
-    AngularEntity {
+    ServerEntity {
```

Comment:
AngularEntity should be added back.
Backend Blueprints can define authorities based on the backend implementation.

Reply:
So backend blueprint uses `entityAngularAuthorities` and `entityAngularReadAuthorities`...
Hmmmm, Ok, I'll use the same 'Partial'  trick as for application.